app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
   def index
+    current_user = User.find_by_id(session[:current_user_id])
```

Comment:
Ok so this is in the `#index` method....will we want to implement this site wide? and make it an instance variable? So it's available on a successful Auth?